.github/compatibility-test-matrices/release-v2.5.x/transfer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "chain-a": ["release-v2.5.x", "release-v6.0.x", "release-v5.1.x", "v5.0.1", "release-v4.2.x", "v4.1.1", "release-v3.4.x", "v3.3.1", "v2.4.2"],
+  "chain-b": ["release-v2.5.x", "release-v6.0.x", "release-v5.1.x", "v5.0.1", "release-v4.2.x", "v4.1.1", "release-v3.4.x", "v3.3.1", "v2.4.2"],
```

Comment:
same comment about not needing to test against all other release branches (just released versions)